### DISCUSSION TITLE
foot_guided_control_base::calc_uの導出過程について

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#51] [Q&A] [answered]
スライドにリンクがある手書きメモの、⑧を求める部分の(i),(ii)の式はどこから出てきたのでしょうか。微分方程式を解く何らかの公式を使用されたのではないかと思ったのですが、その公式へのリンク等、教えていただけますと幸いです。

**Comment by YutaKojio** [comment, marked as answer]
同次(斉次)と非同次に分けて解く，微分方程式の一般的な解きなはずで記憶をたよりに解いてるからなにか見たわけではないけど，
今ぱっと検索して出てくる https://www2.yukawa.kyoto-u.ac.jp/~norihiro.tanahashi/pdf/ODE/note_3.pdf とかかな

**Reply by YutaKojio** [reply]
特解は導関数予想して求めてる
https://www.geisya.or.jp/~mwm48961/kou3/differ_eq3.htm (非同次方程式の特殊解の求め方)
